pkg/localize/locales/en/cluster_kubernetes.en.toml

```
@@ -78,6 +78,13 @@ one = 'Service Account Secret "{{.Name}}" created successfully'
 [cluster.kubernetes.createTokenSecret.log.info.createFailed]
 one = 'Creation of the "{{.Name}}" secret failed:'
 
+[cluster.kubernetes.createTokenSecret.error.info.unauthorized]
+one = '''
+oc token provided has expired or user is not authorized
```

This is confusing as users are not guaranteed to login with token (they can login using credentials etc.)
```suggestion
User is not authorized to create resources on cluster. User session might timed out. Please login to your cluster again using oc or kubectl
```